idoit-install

```diff
@@ -1396,6 +1431,13 @@ function secureMariaDB {
         -e"UPDATE mysql.user SET Password=PASSWORD('${MARIADB_SUPERUSER_PASSWORD}') WHERE User='${MARIADB_SUPERUSER_USERNAME}';" || \
         abort "SQL statement failed"
 
+    log "Set plugin mysql_native_password for $MARIADB_SUPERUSER_USERNAME"
```

Comment:
You can combine the SQL query with the previous one.

Does this change solve the problem with Debian 10?

Reply:
Combined now.
will be added in next commit.
Yeah, this will solve the Debian 10 issue.